docs/Antora/modules/howto_guides/pages/howto_configure_instance.adoc

```diff
@@ -207,28 +237,32 @@ Cpp::
 #include <opendaq/opendaq.h>
 #include <iostream>
 
-class CustomSchedulerImpl : public ImplementationOfWeak<IScheduler>
-{
-    CustomSchedulerImpl(size_t workers)
-    {
-        // initializing
-    }
-    // implementation
-} 
-
 int main(int argc, const char* argv[])
 {
-    daq::SchedulerPtr scheduler = daq::SchedulerPtr(CustomSchedulerImpl(2));
+    daq::LoggerPtr logger = Logger();
+    daq::SchedulerPtr scheduler = daq::Scheduler(logger, 4);
     daq::InstanceBuilderPtr instanceBuilder = daq::InstanceBuilder().setScheduler(scheduler);
     daq::InstancePtr instance = instanceBuilder.build();
 
     return 0;
 }
 ----
+Python::
++
+[source,python]
+----
+import opendaq
+
+logger = opendaq.Logger(opendaq.List(), opendaq.LogLevel.Warn)
+scheduler = opendaq.Scheduler(logger, 4)
+instance_builder = opendaq.InstanceBuilder()
+instance_builder.scheduler = scheduler
+instance = instance_builder.build()
+----
 ====
 
-== Configure openDAQ(TM) Root Device
-If, after creating the Instance object, the developer has not set the root device, the Instance will use the Client device as the root. Using the Instance Builder, developers can discover and connect to available devices using the method `getAvailableDevices`. In the Instance Builder, developers can forcibly set the available device.
+== Configure openDAQ(TM) Default Root Device
+The Instance has the client device as the default root device. A developer can modify the default device by setting the default root device info and local id in the instance builder.
```

Comment:
I'd add here that my setting a new default root device via the connection string, on Instance build, the module manager will be used to create/connect to the device with the given connection string. Said device will then be set as the new root device.

Reply:
this is written in "Configure openDAQ(TM) Root Device"